### DISCUSSION TITLE
Integration: how to add another counter script to retype website?

### DISCUSSION THREAD

**Discussion by sergeykorobeynikov** [retypeapp/retype#475] [Ideas]
So, I want add [Yandex.Metrika](url=https://metrika.yandex.ru/) for my website, I have code:
```html
<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
   (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
   m[i].l=1*new Date();
   for (var j = 0; j < document.scripts.length; j++) {if (document.scripts[j].src === r) { return; }}
   k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
   (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");

   ym(1234567, "init", {
        clickmap:true,
        trackLinks:true,
        accurateTrackBounce:true
   });
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/1234567" style="position:absolute; left:-9999px;" alt="" /></div></noscript>
<!-- /Yandex.Metrika counter -->
```
Is it possible to save this code in a separate file, and in the project settings add files from which you need to integrate the code into the site structure?

**Comment by geoffreymcgill** [comment]
Hi @sergeykorobeynikov 

You can add your script into a custom `_includes/head.html` file.

Add your custom code to a file named `head.html` and add that page into a new `_includes` folder within the root of your project. The code will then be included on all pages of your generated website. 

Additional details available at:
https://github.com/retypeapp/retype/discussions/282#discussioncomment-2343005

Hope this helps.

**Reply by sergeykorobeynikov** [reply]
Thanks, it works!
Unfortunately, this is not in the main documentation, but there is a link to this forum. Better link to #282.
Regards!
